src/internal/Esp32_i2s.c

```diff
@@ -454,8 +434,24 @@ void i2sInit(uint8_t bus_num,
     i2s->pdm_conf.tx_pdm_en = 0;
 #endif
 
-    i2sSetSampleRate(bus_num, sample_rate, bits_per_sample);
-
+    
+	
+
+	if (bits_per_sample == 8 || bits_per_sample == 16)
+	{		
+		i2sSetSampleRate(bus_num, sample_rate, bits_per_sample);
+		i2s->fifo_conf.val = 0;
+		i2s->fifo_conf.rx_fifo_mod_force_en = 1; 
+		i2s->fifo_conf.tx_fifo_mod_force_en = 1; // HN
+		i2s->fifo_conf.rx_fifo_mod = 1; // HN
+		i2s->fifo_conf.tx_fifo_mod = 1;
+		i2s->fifo_conf.rx_data_num = 32; //Thresholds. 
+		i2s->fifo_conf.tx_data_num = 32;
+	}
+	else
```

Comment:
no need for this else, remove from above branch and just put the single call before the branch.

Reply:
OK, changed. It was leftover from my attempts to modify the clock for parallel modes.